### PR TITLE
Added YouTube Room

### PR DESCRIPTION
**Pull request by theflametrooper**

### CODE REVIEW

File: data/RoomData.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ var camperBotChatRooms = [
     'FreeCodeCamp/java',
     'FreeCodeCamp/linux',
     'FreeCodeCamp/LiveCoding',
+    'FreeCodeCamp/YouTube',
```
Comment (abhisekp) [inline data/RoomData.js]: Please sort it lexicographically.

**Comment by ltegman** [comment]
Why was this added and merged? I thought we only add Camperbot to the official chatrooms? Also, you shouldn't be merging your own commits.

**Comment by theflametrooper** [comment]
@ltegman Yeh, sorry my fault! @raisedadead has told me all about that! We are moving from twitch to youtube. This is an official room, maybe replacing the livecoding room, we're not sure yet.

**Comment by ltegman** [comment]
Cool, cool 👍

**Comment by theflametrooper** [comment]
@ltegman I'll make sure it doesn't happen again!

**Comment by BerkeleyTrue** [comment]
@theflametrooper Please revert this PR. Per @QuincyLarson 's response in the core team room, this is not an official chat room.

**Pull request merged**